feature/container/containerz/tests/container_lifecycle/containerz_test.go

```diff
@@ -65,5 +93,155 @@ func TestDeployAndStartContainer(t *testing.T) {
 	if _, err = cntrCli.Ping(ctx, &cpb.PingRequest{}); err != nil {
 		t.Errorf("unable to reach cntrsrv: %v", err)
 	}
+}
+
+// TestRetrieveLogs implements CNTR-1.2 validating that logs can be retrieved from a
+// running container.
+func TestRetrieveLogs(t *testing.T) {
+	ctx := context.Background()
+	cli := startContainer(ctx, t)
+	defer stopContainer(ctx, t, cli)
+
+	logCh, err := cli.Logs(ctx, instanceName, false)
+	if err != nil {
+		t.Errorf("unable to obtain logs for %s: %v", instanceName, err)
+	}
+
+	var logs []string
+	for msg := range logCh {
+		logs = append(logs, msg.Msg)
+		if msg.Error != nil {
+			t.Errorf("logs returned an error: %v", err)
+		}
+	}
+
+	if len(logs) == 0 {
```

Comment:
I presume that the container here is specified to create some logs so there's no chance that this is a flake?